CONTRIBUTING_DESIGN.md

```diff
@@ -0,0 +1,200 @@
+# `/docs` & 11ty documentation
+
+The Red Hat Digital Design System site is built with [Eleventy](https://www.11ty.io/).
+
+## Installation
+Make sure you have at least Node 16 and run:
+
+```shell
+npm ci
+```
+
+## Running Local Server
+```
+npm run watch:docs
+```
+
+This will start eleventy and a watch process that will reload the site if `*.njk`, `*.md`, `*.scss`, or `*.js` files are updated.
+
+## Publishing
+
+Publish to Github Pages
```

Comment:
Would this be on a local branch, publishing to production?

Reply:
I think we can take this out, since we publish via netlify now